WinCCOA_QualityChecks/panels/WinCCOA_QualityChecks/general/Settings.pnl

```diff
@@ -509,79 +510,84 @@ void initTree()
 void fillWithData()
 {
   settings = QgSettings::getAllSettingsAsMapping();
-  
+
   for (int i = 1; i <= dynlen(settings); i++)
   {
     mapping json_data = settings[i];
-    
+
     if (mappingHasKey(json_data, \"settingsVersion\") == false ||
         mappingHasKey(json_data, \"checks\") == false )
     {
       continue;
     }
-    
+
     //determine machting check for id
     dyn_mapping checks = json_data[\"checks\"];
-    
-    
+
+
     for (int j = 1; j <= dynlen(checks); j++)
     {
-      mapping check = checks[j];      
-      
+      mapping check = checks[j];
+
       if ( mappinglen(check) < 1 )
       {
         continue;
-      }    
-    
+      }
+
       Mapping map = Mapping(check);
       dyn_string id_parts = strsplit(map.getAt(\"id\"), \".\");
       string id = \"\";
-      string parent_id = \"\"; 
-      
+      string parent_id = \"\";
+
       for (int z = 1; z <= dynlen(id_parts); z++)
-      {      
+      {
         if (id != \"\")
-        {     
+        {
           id += \".\";
         }
-        
-        id += id_parts[z];    
-        
+
+        id += id_parts[z];
+
         if (z > 1)
         {
           if (parent_id != \"\")
-          {     
+          {
             parent_id += \".\";
           }
           parent_id += id_parts[z-1];
         }
-        
+
         if (!this.itemExists(id))
         {
           this.appendItem(parent_id, id, id_parts[z]);
         }
-      }      
-      
+      }
+
       //read settings depeding on the version
+      anytype lang = getActiveLangId();
+      string sLang = lang == OaLanguage::de_AT || lang == OaLanguage::de_AT_iso88591
```

Comment:
Why not getting the name of the language with getLocale(lang) and checking startsWith("de_") ? => de_AT.utf8 and en_US.utf8 otherwise.